src/mixins/filing-template-mixin.ts

```diff
@@ -340,6 +338,14 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
       }
     }
 
+    // Apply business address changes to filing
+    if (this.officeAddressesChanged) {
+      filing.changeOfRegistration.businessAddress = {
+        mailingAddress: this.getOfficeAddresses.registeredOffice.mailingAddress,
+        deliveryAddress: this.getOfficeAddresses.registeredOffice.deliveryAddress
+      }
```

Comment:
Oh, you're using AddressesIF to store the business address into the registeredOffice object?

Reply:
You bet, this was the api restrictions i referred to above. 
The api is/will be returning these addresses wrapped regardless of EntityType at the moment. 
As our Firm schema refers to these addresses wrapped as businessAddress we need to handle that.

Reply:
Got it.

I went the other way and created [registration business addresses](https://github.com/bcgov/business-create-ui/blob/main/src/interfaces/store-interfaces/state-interfaces/registration-state-interface.ts#L29).

Closing this conversation...